doc/source/building/index.rst

```diff
@@ -376,6 +376,37 @@ interface is self-documenting, so please see ``python dev.py --help`` and
 ``python dev.py <subcommand> --help`` for detailed guidance.
 
 
+.. admonition:: IDE support & editable installs
+
+    While the ``dev.py`` interface is our recommended way of working on SciPy,
+    it has one limitation: because of the custom install location, setting
```

Comment:
Without some trouble, it's also not easy to execute scripts using the "play" button of an IDE or use the IDE to run tests. (I forget whether I was ever successful.) I would suggest generalizing this statement of the limitation, maybe "because of the custom install location, SciPy installed using `dev.py` will only work via `dev.py`; for example, it will not be recognized within an IDE."